course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+# EEB430H1 Theoretical Ecology [24L, 12P]
```

Comment:
Should we include the RQM title bit? Like "... Ecology and Reproducible Quantitative Methods using R" or some such?